### PR TITLE
Small refactoring

### PR DESCRIPTION
**Pull request by vpop**
Some import optimizations and a couple other changes.

### CODE REVIEW

File: src/api/auth_backends.py [hunk 1]
```diff
@@ -158,5 +158,5 @@ def authenticate(self, request, *args, **kwargs):
             pass
 
         if jwt_token:
-            return super().authenticate(request, *args, **kwargs)
+            return super().authenticate(request)
```
Comment (saanobhaai) [inline src/api/auth_backends.py:161]: Why remove `*args` and `**kwargs`?

Comment (vpop) [reply src/api/auth_backends.py:161]: The [`authenticate`](https://github.com/data-mermaid/mermaid-api/blob/master/src/api/auth_backends.py#L31) method signature from `JWTAuthentication` parent class does not define those, unless I am missing something.

**Comment by vpop** [comment]
My fork exists only to be able to submit PRs to the main mermaid-api repo. I am not affiliated with any organization related to MERMAID. I am simply trying to contribute to the MERMAID platform in any way I can. I heard about it from a WCS Ocean tweet. Honored to be the first non-core-team contributor by the way. :grin:

File: src/api/ingest/utils.py [hunk 2]
```diff
@@ -11,11 +8,9 @@
     FishBeltCSVSerializer,
 )
 from api.models import (
-    BENTHICLIT_PROTOCOL,
```
Comment (saanobhaai) [inline src/api/ingest/utils.py:14]: We need to leave these two imports in. They're not used currently, but they're about to be.

File: src/api/auth_backends.py (hunk 1 above)
Comment (saanobhaai) [reply src/api/auth_backends.py:161]: Good point.

**Comment by saanobhaai** [comment]
> My fork exists only to be able to submit PRs to the main mermaid-api repo. I am not affiliated with any organization related to MERMAID. I am simply trying to contribute to the MERMAID platform in any way I can. I heard about it from a WCS Ocean tweet. Honored to be the first non-core-team contributor by the way. 😁

Wonderful! The cleanups are appreciated. Let me know if you have time and interest for more, or feel free to keep cleaning and optimizing.

File: src/api/ingest/utils.py (hunk 2 above)
Comment (vpop) [reply src/api/ingest/utils.py]: Sounds good, committed.

**Comment by vpop** [comment]
> > My fork exists only to be able to submit PRs to the main mermaid-api repo. I am not affiliated with any organization related to MERMAID. I am simply trying to contribute to the MERMAID platform in any way I can. I heard about it from a WCS Ocean tweet. Honored to be the first non-core-team contributor by the way. grin
> 
> Wonderful! The cleanups are appreciated. Let me know if you have time and interest for more, or feel free to keep cleaning and optimizing.

Sure, and yes, I have both time and interest to contribute in a more organized fashion if there is a need.

**Pull request merged**